
错误信息开始=====>
错误类型:ChangeConflictException
错误信息:找不到行或行已更改。
错误堆栈:
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 BLL.Funs.SubmitChanges() 位置 D:\DepCode\CNCEC\SUBHSSE\SUBHSSE\BLL\Common\Funs.cs:行号 860
出错时间:11/12/2019 23:15:14
出错时间:11/12/2019 23:15:14


错误信息开始=====>
错误类型:ChangeConflictException
错误信息:找不到行或行已更改。
错误堆栈:
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 BLL.Funs.SubmitChanges() 位置 D:\DepCode\CNCEC\SUBHSSE\SUBHSSE\BLL\Common\Funs.cs:行号 860
出错时间:11/12/2019 23:15:14
出错时间:11/12/2019 23:15:14


错误信息开始=====>
错误类型:ChangeConflictException
错误信息:找不到行或行已更改。
错误堆栈:
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 BLL.Funs.SubmitChanges() 位置 D:\DepCode\CNCEC\SUBHSSE\SUBHSSE\BLL\Common\Funs.cs:行号 860
出错时间:11/12/2019 23:15:14
出错时间:11/12/2019 23:15:14


错误信息开始=====>
错误类型:ChangeConflictException
错误信息:找不到行或行已更改。
错误堆栈:
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 BLL.Funs.SubmitChanges() 位置 D:\DepCode\CNCEC\SUBHSSE\SUBHSSE\BLL\Common\Funs.cs:行号 860
出错时间:11/12/2019 23:15:14
出错时间:11/12/2019 23:15:14


错误信息开始=====>
错误类型:ChangeConflictException
错误信息:找不到行或行已更改。
错误堆栈:
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 BLL.Funs.SubmitChanges() 位置 D:\DepCode\CNCEC\SUBHSSE\SUBHSSE\BLL\Common\Funs.cs:行号 860
出错时间:11/12/2019 23:15:14
出错时间:11/12/2019 23:15:14


错误信息开始=====>
错误类型:ChangeConflictException
错误信息:找不到行或行已更改。
错误堆栈:
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 BLL.Funs.SubmitChanges() 位置 D:\DepCode\CNCEC\SUBHSSE\SUBHSSE\BLL\Common\Funs.cs:行号 860
出错时间:11/12/2019 23:15:14
出错时间:11/12/2019 23:15:14


错误信息开始=====>
错误类型:ChangeConflictException
错误信息:找不到行或行已更改。
错误堆栈:
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 BLL.Funs.SubmitChanges() 位置 D:\DepCode\CNCEC\SUBHSSE\SUBHSSE\BLL\Common\Funs.cs:行号 860
出错时间:11/12/2019 23:15:47
出错时间:11/12/2019 23:15:47


错误信息开始=====>
错误类型:ChangeConflictException
错误信息:找不到行或行已更改。
错误堆栈:
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 BLL.Funs.SubmitChanges() 位置 D:\DepCode\CNCEC\SUBHSSE\SUBHSSE\BLL\Common\Funs.cs:行号 860
出错时间:11/12/2019 23:15:48
出错时间:11/12/2019 23:15:48


错误信息开始=====>
错误类型:ChangeConflictException
错误信息:找不到行或行已更改。
错误堆栈:
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 BLL.Funs.SubmitChanges() 位置 D:\DepCode\CNCEC\SUBHSSE\SUBHSSE\BLL\Common\Funs.cs:行号 860
出错时间:11/12/2019 23:17:17
出错时间:11/12/2019 23:17:17


错误信息开始=====>
错误类型:ChangeConflictException
错误信息:找不到行或行已更改。
错误堆栈:
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 BLL.Funs.SubmitChanges() 位置 D:\DepCode\CNCEC\SUBHSSE\SUBHSSE\BLL\Common\Funs.cs:行号 860
出错时间:11/12/2019 23:17:17
出错时间:11/12/2019 23:17:17


错误信息开始=====>
错误类型:ChangeConflictException
错误信息:找不到行或行已更改。
错误堆栈:
   在 System.Data.Linq.ChangeProcessor.SubmitChanges(ConflictMode failureMode)
   在 System.Data.Linq.DataContext.SubmitChanges(ConflictMode failureMode)
   在 BLL.Funs.SubmitChanges() 位置 D:\DepCode\CNCEC\SUBHSSE\SUBHSSE\BLL\Common\Funs.cs:行号 860
出错时间:11/12/2019 23:17:53
出错时间:11/12/2019 23:17:53

